apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -397,7 +434,8 @@ export default function RubricContainer({
               tabSelectCallback={tabSelectCallback}
               reportingData={reportingData}
               allTeacherEvaluationData={allTeacherEvaluationData}
-              allAiEvaluationStatus={allAiEvaluationStatus}
+              aiEvalStatusCounters={aiEvalStatusCounters}
+              updateAiEvalStatusMap={updateAiEvalStatusMap}
```

Comment:
Wouldn't this work just as well and remove the need for the wrapper function above?

```suggestion
              updateAiEvalStatusMap={setAiEvalStatusMap}
```